manifests/configmap-seed/files/restart-mtls.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "Restarting mtls process in main container"
+
+killall uwsgi
```

Comment:
The start-up time on mtls can be rather large. we should look at doing this as part of ether the main service or via a sidecar.

Reply:
How large is large? Maybe it's acceptable for the use case. It only needs to be available the moment someone requests a cert, right? And that happens maybe once a day given the 18h timeout.

Reply:
Consider, also, the problem I'm trying to solve here: Currently, if the user list is updated, the Pod must be manually restarted in order to update the server. With this change, the Pod restarts itself. It's objectively better, even if it's not perfect.